test/forms/mappings/SelectLocalDateFormatterSpec.scala

```diff
@@ -143,40 +156,52 @@ class SelectLocalDateFormatterSpec extends SpecBase {
     val day: Int = 1
 
     val key = "start"
-    val invalidMsgKey = "cf.form.error.start.date-number-invalid"
-    val monthMsgKey = "cf.form.error.start.date.invalid.month"
-    val yearMsgKey = "cf.form.error.start.date.invalid.year"
-    val invalidRealDateMsgKey = "cf.form.error.start.date.invalid.real-date"
+    val emptyMonthAndYearMsgKey = "cf.cash-account.transactions.request.start.date.empty.month.year"
```

Comment:
Should we wrap these messages with `messages()` for localization?

Reply:
No @Neozxc, as we are only dealing and checking the message keys' name and the localization will take place in the view or in the controller (if used)